assets/css/global.css

```
@@ -104,7 +106,7 @@
 /* Footer CSS */
 
 .footer {
-    background-image: url('../../assets/images/spacebg.jpg');
+    background-image: linear-gradient(rgba(21, 61, 165, .17), rgba(21, 61, 165, .17)), url("../../assets/images/spacebg.jpg");
```

```suggestion
    background-image: linear-gradient(rgba(21, 61, 165, .17), rgba(21, 61, 165, .17)), url("../images/spacebg.jpg");
```

another fix for this path, I think this makes more sense